core/crypto/sub.mk

```diff
@@ -22,6 +22,9 @@ endif
 ifneq ($(CFG_CRYPTO_CTS_FROM_CRYPTOLIB),y)
 srcs-$(CFG_CRYPTO_CTS) += aes-cts.c
 endif
+ifeq ($(filter y,$(CFG_CRYPTO_SM2_PKE) $(CFG_CRYPTO_SM2_KEP),y),y)
```

Comment:
This fails when `CFG_CRYPTO_SM2_PKE=n` and `CFG_CRYPTO_SM2_KEP=y`. please crosscheck.
Looks like the issue was already there.

Reply:
Yes, good catch, that's a bug in 354d31d46. I'll fix.